server/helpers/parseTxBody.js

```diff
@@ -1,6 +1,6 @@
-const cbor = require('borc') // TODO parse bignumbers properly
+const cbor = require('cbor') // TODO parse bignumbers properly
 
-const coinsFromOutputAmount = (outputAmount) =>
+const coinsFromOutputValue = (outputAmount) =>
```

Comment:
Since we're changing the function name here I'd also rename the function parameter to outputValue

Reply:
fixed :+1: